infra/main.parameters.json

```diff
@@ -23,6 +23,18 @@
     "openAiSkuName": {
       "value": "S0"
     },
+    "computerVisionServiceName": {
+      "value": "${AZURE_COMPUTER_VISION_SERVICE}"
+    },
+    "computerVisionResourceGroupName": {
+      "value": "${AZURE_COMPUTER_VISION_RESOURCE_GROUP}"
+    },
+    "computerVisionResourceGroupLocation": {
+      "value": "${AZURE_COMPUTER_VISION_LOCATION}"
```

Comment:
I think this one needs to default to eastus, as thats what main.bicep has currently, and then main.bicep should be set to nothing with a comment "// Set in main.parameters.json"

Reply:
AI Vision and Document Intelligence is now widely available, but I'll set the default as you suggest. 
https://azure.microsoft.com/en-gb/explore/global-infrastructure/products-by-region/?products=cognitive-services